ecl/eclccserver/eclccserver.cpp

```diff
@@ -900,6 +900,13 @@ int main(int argc, const char *argv[])
         const char *wuid = globals->queryProp("@workunit");
         if (wuid)
         {
+#ifdef _CONTAINERIZED
+            CSDSServerStatus serverstatus("ECLCCserverThread");
+            serverstatus.queryProperties()->setProp("@cluster", globals->queryProp("@name"));
+            serverstatus.queryProperties()->setProp("WorkUnit", wuid);
+            serverstatus.commitProperties();
```

Comment:
I don't think this block should be required.

Each active compile thread already has a serverStatus in it for the workunit it's dealing with.
When none are compiling, there are 0.

Because of that and the Thor case, I think this code:

I think the : 
```
        if (!smcQueue->foundQueueInStatusServer)
            targetCluster->clusterStatusDetails.appendf("Cluster %s not listening for workunits; ", clusterName.str());
```

Needs looking at or disabling in container mode.

Does this also impact hthor agents when no hthor jobs are running?

@wangkx